src/server/scripts/Northrend/zone_dalaran.cpp

```diff
@@ -795,6 +796,57 @@ class npc_dalaran_warrior : public CreatureScript
     };
 };
 
+enum ToyPlane
+{
+    NPC_DND_DALARAN_TOY_STORE_PLANE_STRING_HOOK = 29807,
+
+    SPELL_TOY_PLANE_CABLE   = 55281,
+};
+
+struct npc_cosmetic_toy_plane : public ScriptedAI
+{
+    npc_cosmetic_toy_plane(Creature* creature) : ScriptedAI(creature)
+    {
+        Movement::MoveSplineInit init(me);
```

Comment:
Please move this to Reset(), avoid calling code inside the constructor (stuff may not be initialized at that point)